Brokerages/GDAX/GDAXBrokerage.cs

```diff
@@ -290,6 +295,125 @@ public override List<Cash> GetCashBalance()
 
             return list;
         }
+
+        /// <summary>
+        /// Gets the history for the requested security
+        /// </summary>
+        /// <param name="request">The historical data request</param>
+        /// <returns>An enumerable of bars covering the span specified in the request</returns>
+        public override IEnumerable<BaseData> GetHistory(HistoryRequest request)
+        {
+            // GDAX API only allows us to support history requests for TickType.Trade
+            if (request.TickType != TickType.Trade)
+            {
+                yield break;
```

Comment:
Should we log/emit a message here?

Reply:
I think it should `yield break` silently because `QCAlgorithm.History` will automatically generate history requests for all tick types.